lib/tasks_private/terraform/provision/variables.tf

```diff
@@ -16,8 +15,7 @@ variable "vm_name" {
 
 variable "power_instance_id" {
     description = "Power Virtual Server instance ID associated with your IBM Cloud account (note that this is NOT the API key)"
-    #default = "8fa27c40-827c-4568-8813-79b398e9cd27"
-    default = "749e3492-1ff4-4d45-b43c-513674930661"
+    default =  "888f9bf2-ab12-48c4-9dc5-8da65de951a8"
```

Comment:
Same comment re: should this be hard-coded to this

Reply:
I was building it without the rake wrapper. These hardcoding are left overs. I can. remove them.